### PR TITLE
configure typescript

### PR DESCRIPTION
**Pull request by pashchuk**

### CODE REVIEW

**Comment by pashchuk** [comment]
[Setup typescript compiler and envorinment](https://trello.com/c/8LJyI99C/5-setup-typescript-compiler-and-envorinment)

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+        "sourceMap": true,
+        "noImplicitAny": true,
+        "module": "commonjs",
+        "target": "es5",
+        "jsx": "react"
+    },
+    "include": [
+        "./src/**/*"
+    ]
+}
```
Comment (ytalashko) [inline tsconfig.json]: Please, add newline ending.

Comment (pashchuk) [reply tsconfig.json]: done

**Review by ytalashko — APPROVED** [review]

**Comment by ytalashko** [comment]
LGTM

**Pull request merged**